### DISCUSSION TITLE
Is there a feature to download images from chat in the form of photos?

### DISCUSSION THREAD

**Discussion by harisvivi20** [flyerhq/flutter_chat_ui#662] [Ideas & Requests]
Is there a feature to download images from chat in the form of photos?

or is there another way to be able to download the image of the photo sent from the chat?

**Comment by demchenkoalex** [comment]
I think this would be outside of the scope for this package - I know `v1` is not a most complete solution, and I am working on `v2` which will handle photos much more elegant (with cross platform cache, blurred preview, download progress etc). But I do not plan to add a download to gallery feature, since it is probably something you would put into a menu when you long press on a message. I will try to provide a menu as part of `v2` as well, but for now you just have press event callbacks, and then if you do a menu and download button there, you could use any package to save it to the gallery, like this one https://pub.dev/packages/image_gallery_saver
